price_security/views/product_pricelist_views.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
```

Comment:
@zaoral this it's not necessary. This show the field in any place. Like see in this image:
![seleccion_454](https://user-images.githubusercontent.com/7657373/41035949-b463f742-6964-11e8-95aa-587ed2247ae5.png)